### PR TITLE
[14.0][MIG] account_tax_balance

### PR DESCRIPTION
**Pull request by ozono**
account_tax_balance migration

### CODE REVIEW

**Comment by fredzamoabg** [comment]
It looks good!

**Review by fredzamoabg — APPROVED** [review]

**Comment by jado95** [comment]
Hi, i have an error when try to install the module. The error is:

File "/account-financial-reporting/account_tax_balance/hooks.py", line 18, in pre_init_hook
    "`ALTER TABLE account_move ADD COLUMN IF NOT EXISTS financial_type VARCHAR`"

psycopg2.ProgrammingError: ERROR:  syntax error at or near "NOT"
`LINE 1: ALTER TABLE account_move ADD COLUMN IF NOT EXISTS financial_`

**Comment by pedrobaeza** [comment]
@jado95 it seems you have a very outdated PG version

**Comment by jado95** [comment]
> @jado95 it seems you have a very outdated PG version

It's true. I change query for my version and install the module :+1:

**Comment by pedrobaeza** [comment]
You will have problems not only here, but across all Odoo. You should upgrade your PG version for using it with v14.

**Comment by jado95** [comment]
> You will have problems not only here, but across all Odoo. You should upgrade your PG version for using it with v14.

Of course, i update PG to v12.
Thank's

**Review by AntoniRomera — APPROVED** [review]

**Review by jado95 — APPROVED** [review]

**Comment by mileo** [comment]
Could you check the image on the readme?

 https://github.com/ozono/account-financial-reporting/tree/14.0-mig-account_tax_balance/account_tax_balance

**Comment by ozono** [comment]
> Could you check the image on the readme?
> 
> https://github.com/ozono/account-financial-reporting/tree/14.0-mig-account_tax_balance/account_tax_balance

Done

File: account_tax_balance/README.rst
Comment (tafaRU) [inline account_tax_balance/README.rst:44]: If needed, this change has to be applied in readme/USAGE.rst file, otherwise further changes will discard yours by OCA Bot (see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst for further info).

File: account_tax_balance/README.rst
Comment (tafaRU) [inline account_tax_balance/README.rst:76]: Please apply this change in readme/CONTRIBUTORS.rst as explained in the previous comment.

**Review by tafaRU — CHANGES_REQUESTED** [review]
Thanks for you PR, @ozono!

In addition to my review comments, please also revert changes related to translation files (.po, .pot) as written in https://odoo-community.org/blog/the-oca-blog-1/post/translations-platform-81 

Except for this, overall LGTM.

File: account_tax_balance/README.rst
Comment (ozono) [reply account_tax_balance/README.rst:44]: Done. Thanks.

File: account_tax_balance/README.rst
Comment (ozono) [reply account_tax_balance/README.rst:76]: It was already there

File: account_tax_balance/readme/USAGE.rst
Comment (pedrobaeza) [inline account_tax_balance/readme/USAGE.rst]: I'm afraid this change is not correct. It should be reverted to the previous one, with relative path, and build with the README generator in an absolute way, but only accessible once this is merged. Sorry for all the confusion around. If you think it's too much to do, we can merge it this way and then I'll do direct commit for reverting back.

File: account_tax_balance/README.rst
Comment (tafaRU) [reply account_tax_balance/README.rst:76]: Ok, I missed it!

File: account_tax_balance/readme/USAGE.rst
Comment (ozono) [reply account_tax_balance/readme/USAGE.rst]: No problem. Done.

Comment (pedrobaeza) [reply account_tax_balance/readme/USAGE.rst]: I don't see the relative path, but I'm fixing after the merge.

**Review by tafaRU — APPROVED** [review]

**Pull request merged**
